libs/novui/src/recipes/toggle.recipe.ts

```diff
@@ -0,0 +1,40 @@
+import { SwitchStylesNames } from '@mantine/core';
+import { defineSlotRecipe } from '@pandacss/dev';
+
+// full enumeration of the component library's slots
+const SLOTS: SwitchStylesNames[] = [
+  'root',
+  'track',
+  'trackLabel',
+  'thumb',
+  'input',
+  'body',
+  'labelWrapper',
+  'label',
+  'description',
+  'error',
+];
+
+export const TOGGLE_RECIPE = defineSlotRecipe({
+  className: 'toggle',
+  jsx: ['Toggle'],
+  slots: SLOTS,
+  base: {
+    track: {
+      borderRadius: 'pill',
+      backgroundColor: 'toggle.background',
+      border: 'none',
+      height: 150,
+      width: 250,
+      'input:checked + &': {
+        backgroundColor: 'toggle.checked',
+      },
+    },
+    thumb: {
+      backgroundColor: 'toggle.thumb',
+      border: 'none',
+      height: 125,
+      width: 125,
+    },
+  },
```

Comment:
```suggestion
  },
  staticCss: ['*'],
```
We can optimize the CSS later, but this ensures that we don't have any frustrating issues with styles not generating